libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts

```diff
@@ -265,6 +273,15 @@ export class SkyColorpickerComponent implements OnInit, OnDestroy {
     return this.#_colorpickerRef;
   }
 
+  protected get inputId(): string | undefined {
+    return this.#_inputId;
+  }
+
+  protected set inputId(value: string | undefined) {
+    this.#_inputId = value;
+    this.#changeDetector.markForCheck();
```

Comment:
The `inputId` is only ever set by the `SkyColorpickerInputService`, so you _could_ simplify this by removing the getter/setter and calling `markForCheck` in the subscribe block.
```
protected inputId: string | undefined;

this.#colorpickerInputSvc.inputId
  .pipe(takeUntil(this.#ngUnsubscribe))
  .subscribe((id) => {
    this.inputId = id;
    this.#changeDetector.markForCheck();
  });
```